### PR TITLE
Plugins

### PR DESCRIPTION
**Pull request by arboss**
added descriptions for eightball feature

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,9 @@
 __pycache__/
 *.py[cod]
 
+# Editor tmp-files
+*.swp
+
```
Comment (jkowalleck) [inline .gitignore]: unnecessary change

@arboss 
-> this should be handled by your personal global `gitignore` - this is not project-specific .... 
see http://technicalpickles.com/posts/configure-git-to-globally-ignore-some-files/

Comment (arboss) [inline .gitignore]: :+1:
